oaknational/Oak-Web-Application
# [CUR-436] "thanks for downloading" `<SuccessMessage />` component

## Description

Music year: 1996

- Adds a new general purpose `<SuccessMessage/>` component for displaying success

## Issue(s)

Implements component part of `CUR-436`

## How to test

1. Go to storybook preview build
2. Checkout `/?path=/story/components-curriculumcomponents-successmessage--success-message`
3. You should see whats applied in the screenshots 

## Screenshots

Small
<img width="322" alt="Screenshot 2024-04-23 at 15 06 58" src="https://github.com/oaknational/Oak-Web-Application/assets/235915/3b1a41c7-7796-4bed-8442-7047a29b6f95">

Large

<img width="836" alt="Screenshot 2024-04-23 at 15 07 25" src="https://github.com/oaknational/Oak-Web-Application/assets/235915/858ed212-c171-4291-b886-e6254d2312a3">

## Checklist
